server/cluster/cluster_worker.go

```diff
@@ -196,6 +196,16 @@ func (c *RaftCluster) HandleReportSplit(request *pdpb.ReportSplitRequest) (*pdpb
 	originRegion := proto.Clone(right).(*metapb.Region)
 	originRegion.RegionEpoch = nil
 	originRegion.StartKey = left.GetStartKey()
+
+	for _, region := range []*metapb.Region{left, right} {
+		regionInfo := core.NewRegionInfo(region, nil)
+		if err = c.HandleRegionHeartbeat(regionInfo); err != nil {
+			log.Warn("region handler error",
+				zap.Stringer("region-meta", region),
+				errs.ZapError(err))
+		}
+	}
+
```

Comment:
I think some metrics are needed to see how many regions and the splitting event is handled here.

Reply:
I think the metrics can see grpc metric in pd.